week7/PhotoInspiration/app/src/main/java/com/manuflowers/photoinspiration/ui/home/HomeViewModelFactory.kt

```diff
@@ -0,0 +1,16 @@
+package com.manuflowers.photoinspiration.ui.home
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.manuflowers.photoinspiration.application.PhotoInspirationApplication
+
+class HomeViewModelFactory:ViewModelProvider.Factory  {
+    override fun <T : ViewModel?> create(modelClass: Class<T>): T {
+        if (modelClass.isAssignableFrom(HomeViewModel::class.java)) {
```

Comment:
You did this here, so I'd love to have seen you pass in the repository instead of the application or anything else that is bound to Android.

Reply:
Done!